solr/solrj-streaming/src/java/org/apache/solr/client/solrj/io/stream/expr/StreamExpressionParser.java

```diff
@@ -132,15 +132,12 @@ private static StreamExpressionNamedParameter generateNamedParameterExpression(S
         }
       }
 
-      // If contains ` replace with "
+      // If contains ` replace with ". Preserves ` which are escaped.
```

Comment:
COuld we rework this comment?   NOt sure I understand the ` " .  Preserves` text...?

Reply:
Done